steps/25_split_package/step.md

```diff
@@ -0,0 +1,148 @@
+# Modularisation of sktime into Independent Component Packages
+
+Contributors: @yarnabrina
+
+## Introduction
+
+As the sktime codebase continues to expand, the current monolithic structure is
+presenting increasing challenges in terms of maintenance, dependency management, and
+release coordination. This proposal outlines a plan to modularise sktime into
```

Comment:
would more packages not increase the amount of challenges in release *coordination? Given that it is easier to coordinate (or simply: to carry out) a release for a single package than for 5.